doc/sfsmount.1.adoc

```diff
@@ -162,6 +162,9 @@ disables cache (default: 0).
 *-o readaheadmaxwindowsize=*'KB'::
 Set max value of readahead window per single descriptor in kibibytes (default: 16384).
 
+*-o readcachemaxsize=*'MB'::
```

Comment:
please use camelcase in our documentation.... everywhere.
```suggestion
*-o rearCacheMaxSize=*'MB'::
```

Reply:
Probably this change should be in a separate PR changing all arguments.

Reply:
We better let this for a dedicated PR improving sfsmount doc readability